.github/workflows/dotnet-analyzers.yml

```
@@ -0,0 +1,37 @@
+name: DotNet Analyzers
+
+on:
+  push:
+    branches: [ "main" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)